Library/Formula/poppler.rb

```diff
@@ -52,7 +52,9 @@ def install
       --enable-introspection=yes
     ]
 
-    if build.with? "qt"
+    if build.with?("qt") && build.with?("qt5")
+      odie "poppler: --with-qt and --with-qt5 cannot be used at the same time"
```

Comment:
I think the reason is mostly about consistence. If you look for a reason other than consistence, I think there are some small differences. `odie` will quit program immediately, while `raise` is something you can rescue.